swarm/network/simulation/kademlia.go

```diff
@@ -96,3 +98,106 @@ func (s *Simulation) kademlias() (ks map[enode.ID]*network.Kademlia) {
 	}
 	return ks
 }
+
```

Comment:
ideally we should submit kademlia changes as a separate PR tracking as https://github.com/ethersphere/go-ethereum/issues/1298